airflow/contrib/operators/dataflow_operator.py

```diff
@@ -56,6 +56,7 @@ class DataFlowJavaOperator(BaseOperator):
             'partitionType': 'DAY',
             'labels': {'foo' : 'bar'}
         },
+        gcp_conn_id='google_cloud_default',
```

Comment:
mind replacing 'google_cloud_default' with another connection name? 'google_cloud_default' is the default value for gcp_conn_id. Here and the other place.